app/models/publisher.rb

```diff
@@ -43,6 +43,8 @@ class Publisher < ApplicationRecord
   before_create :build_default_channel
   before_destroy :dont_destroy_publishers_with_channels
 
+  scope :case_insensitive_find_by_email, -> (email_to_find) { where('lower(publishers.email) = :email_to_find', email_to_find: email_to_find.downcase) }
```

Comment:
could we rename the method to `#find_by_email_case_insensitive` ?
it seems like that would be consistent with other finder methods.